packages/@aws-cdk/aws-cloudfront/lib/cache-policy.ts

```diff
@@ -0,0 +1,291 @@
+import { Duration, Resource } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnCachePolicy } from './cloudfront.generated';
+
+/**
+ * Represents a Cache Policy
+ * @experimental
+ */
+export interface ICachePolicy {
+  /**
+   * The ID of the cache policy
+   * @attribute
+   */
+  readonly cachePolicyId: string;
+}
+
+/**
+ * Properties for creating a Cache Policy
+ * @experimental
+ */
+export interface CachePolicyProps {
+  /**
+   * A unique name to identify the cache policy.
+   * The name must only include '-', '_', or alphanumeric characters.
+   */
+  readonly cachePolicyName: string;
```

Comment:
Make this optional by deriving a valid name from `this.node.id` or `this.node.uniqueId`?

Reply:
Yeah, that's a fair idea. Will do!